### ISSUE TITLE
5.2.0 Refuses to start on Mint 19.3

### ISSUE THREAD

**Issue by Johnnyreyman** [minetest/minetest#10081]
##### Minetest version

```
Minetest 5.2.0 (Linux)
Using Irrlicht 1.8.4
BUILD_TYPE=Release
RUN_IN_PLACE=0
USE_GETTEXT=1
USE_SOUND=1
USE_CURL=1
USE_FREETYPE=1
USE_LUAJIT=1
STATIC_SHAREDIR="/usr/share/minetest"
```

##### OS / Hardware

Operating system: Linux Mint 19.3
CPU: AMD A6-5200 APU with Radeon™ HD Graphics × 4

GPU model: Advanced Micro Devices, Inc. [AMD/ATI] Kabini [Radeon HD 8400 / R3 Series] (prog-if 00 [VGA controller])
OpenGL version: 3.1

##### Summary
The game refuses to start via command line or when launching from desktop or application menu.

**Comment by NathanSalapat** [comment]
When starting in a terminal does any output get reported?
I use LinxMint 19.3 and Minetest works just fine.

**Comment by Johnnyreyman** [comment]
No output displays in the terminal.

Jun 20, 2020 7:47:22 PM Nathan Salapat <notifications@github.com>:

**Comment by SmallJoker** [comment]
This looks familiar. Could you please try the steps from here? https://forum.minetest.net/viewtopic.php?f=6&t=24843

 * Run Minetest in `gdb` to check for crashes
 * Run Minetest with `--trace` (and check `~/.minetest/debug.txt`)
 * `menu_clouds = false` in `~/.minetest/minetest.conf`
 * `video_driver = software`

**Comment by Johnnyreyman** [comment]
**Output of gbd**
`(gdb) r
Starting program: /usr/bin/minetest 
[Thread debugging using libthread_db enabled]
Using host libthread_db library "/lib/x86_64-linux-gnu/libthread_db.so.1".
[New Thread 0x7ffff7f94700 (LWP 15959)]
[Thread 0x7ffff7f94700 (LWP 15959) exited]
[New Thread 0x7fffe716c700 (LWP 15960)]
[Thread 0x7fffe716c700 (LWP 15960) exited]
[New Thread 0x7fffe716c700 (LWP 15963)]
AL lib: (EE) ALCpulsePlayback_connectStream: Stream did not get ready: Timeout
[Thread 0x7fffe716c700 (LWP 15963) exited]
2020-06-21 02:30:57: ERROR[Main]: An unhandled exception occurred: Audio: Global Initialization: Context Create
2020-06-21 02:30:57: ERROR[Main]: In thread 7ffff7f98000:
2020-06-21 02:30:57: ERROR[Main]: /build/minetest-dHPfdb/minetest-5.2.0/src/main.cpp:e8: int main(int, char**): A fatal error occurred: Audio: Global Initialization: Context Create

Thread 1 "minetest" received signal SIGABRT, Aborted.
__GI_raise (sig=sig@entry=6) at ../sysdeps/unix/sysv/linux/raise.c:51
51	../sysdeps/unix/sysv/linux/raise.c: No such file or directory.
(gdb) bt full
#0  __GI_raise (sig=sig@entry=6) at ../sysdeps/unix/sysv/linux/raise.c:51
        set = {__val = {4096, 14419211042435378, 140737310452288, 6, 0, 4098, 
            0, 0, 0, 0, 0, 0, 0, 0, 0, 0}}
        pid = <optimized out>
        tid = <optimized out>
        ret = <optimized out>
#1  0x00007ffff4973801 in __GI_abort () at abort.c:79
        save_stage = 1
        act = {__sigaction_handler = {sa_handler = 0x92, sa_sigaction = 0x92}, 
          sa_mask = {__val = {140737307925322, 140737085706136, 
              14156107766117994240, 93825003102016, 140737488342976, 10, 
              93825003102016, 93825003102016, 140737307924059, 93825003101953, 
              93825003102016, 93825003102016, 14156107766117994240, 
              140737310494912, 93825003102016, 93825003102016}}, 
          sa_flags = -180430705, sa_restorer = 0x555555fb1f40 <errorstream>}
        sigs = {__val = {32, 0 <repeats 15 times>}}
        __cnt = <optimized out>
        __set = <optimized out>
        __cnt = <optimized out>
        __set = <optimized out>
#2  0x0000555555b21407 in fatal_error_fn(char const*, char const*, unsigned int, char const*) ()
No symbol table info available.
---Type <return> to continue, or q <return> to quit---
#3  0x000055555568a93f in main ()
No symbol table info available.
`
**Output of minetest --trace**
`2020-06-21 02:35:46: [Main]: Enabling trace level debug output
2020-06-21 02:35:46: INFO[Main]: Using system-wide paths (NOT RUN_IN_PLACE)
2020-06-21 02:35:46: INFO[Main]: Detected share path: /usr/share/minetest
2020-06-21 02:35:46: INFO[Main]: Detected user path: /home/johnathanr/.minetest
2020-06-21 02:35:46: INFO[Main]: Detected cache path: /home/johnathanr/.cache/minetest
2020-06-21 02:35:46: INFO[Main]: Using locale directory /usr/share/minetest/locale
2020-06-21 02:35:46: INFO[Main]: minetest with SER_FMT_VER_HIGHEST_READ=28, BUILD_TYPE=Release
2020-06-21 02:35:46: INFO[Main]: RUN_IN_PLACE=0
2020-06-21 02:35:46: INFO[Main]: USE_GETTEXT=1
2020-06-21 02:35:46: INFO[Main]: USE_SOUND=1
2020-06-21 02:35:46: INFO[Main]: USE_CURL=1
2020-06-21 02:35:46: INFO[Main]: USE_FREETYPE=1
2020-06-21 02:35:46: INFO[Main]: USE_LUAJIT=1
2020-06-21 02:35:46: INFO[Main]: STATIC_SHAREDIR="/usr/share/minetest"
2020-06-21 02:35:46: VERBOSE[Main]: httpfetch_init: parallel_limit=8
2020-06-21 02:35:46: INFO[Main]: Message locale is now set to: LC_CTYPE=en_US.UTF-8;LC_NUMERIC=C;LC_TIME=en_US.UTF-8;LC_COLLATE=en_US.UTF-8;LC_MONETARY=en_US.UTF-8;LC_MESSAGES=en_US.UTF-8;LC_PAPER=en_US.UTF-8;LC_NAME=en_US.UTF-8;LC_ADDRESS=en_US.UTF-8;LC_TELEPHONE=en_US.UTF-8;LC_MEASUREME
2020-06-21 02:35:46: INFO[Main]: NT=en_US.UTF-8;LC_IDENTIFICATION=en_US.UTF-8
2020-06-21 02:35:46: VERBOSE[Main]: Determining world path
2020-06-21 02:35:46: INFO[Main]: Searching worlds...
2020-06-21 02:35:46: INFO[Main]:   In /home/johnathanr/.minetest/worlds: 
2020-06-21 02:35:46: INFO[Main]: Marshland Dolt Islands What the What synolvia-co world What Swan City Synolvia Co Chickens 
2020-06-21 02:35:46: INFO[Main]: 9 found.
2020-06-21 02:35:46: INFO[Main]: Creating default world at [/home/johnathanr/.minetest/worlds/world]
2020-06-21 02:35:46: VERBOSE[Main]: Determining gameid/gamespec
2020-06-21 02:35:46: INFO[Main]: Using world gameid [minetest]
2020-06-21 02:35:46: INFO[Main]: Using commanded world path [/home/johnathanr/.minetest/worlds/world]
AL lib: (EE) ALCpulsePlayback_connectStream: Stream did not get ready: Timeout
2020-06-21 02:37:32: ERROR[Main]: An unhandled exception occurred: Audio: Global Initialization: Context Create
2020-06-21 02:37:32: ERROR[Main]: In thread 7f8a2fa41000:
2020-06-21 02:37:32: ERROR[Main]: /build/minetest-dHPfdb/minetest-5.2.0/src/main.cpp:e8: int main(int, char**): A fatal error occurred: Audio: Global Initialization: Context Create
Aborted (core dumped)
`
**Output of minetest --trace after menu_clouds = false**
`2020-06-21 12:46:17: [Main]: Enabling trace level debug output
2020-06-21 12:46:17: INFO[Main]: Using system-wide paths (NOT RUN_IN_PLACE)
2020-06-21 12:46:17: INFO[Main]: Detected share path: /usr/share/minetest
2020-06-21 12:46:17: INFO[Main]: Detected user path: /home/johnathanr/.minetest
2020-06-21 12:46:17: INFO[Main]: Detected cache path: /home/johnathanr/.cache/minetest
2020-06-21 12:46:17: INFO[Main]: Using locale directory /usr/share/minetest/locale
2020-06-21 12:46:17: INFO[Main]: minetest with SER_FMT_VER_HIGHEST_READ=28, BUILD_TYPE=Release
2020-06-21 12:46:17: INFO[Main]: RUN_IN_PLACE=0
2020-06-21 12:46:17: INFO[Main]: USE_GETTEXT=1
2020-06-21 12:46:17: INFO[Main]: USE_SOUND=1
2020-06-21 12:46:17: INFO[Main]: USE_CURL=1
2020-06-21 12:46:17: INFO[Main]: USE_FREETYPE=1
2020-06-21 12:46:17: INFO[Main]: USE_LUAJIT=1
2020-06-21 12:46:17: INFO[Main]: STATIC_SHAREDIR="/usr/share/minetest"
2020-06-21 12:46:17: VERBOSE[Main]: httpfetch_init: parallel_limit=8
2020-06-21 12:46:17: INFO[Main]: Message locale is now set to: LC_CTYPE=en_US.UTF-8;LC_NUMERIC=C;LC_TIME=en_US.UTF-8;LC_COLLATE=en_US.UTF-8;LC_MONETARY=en_US.UTF-8;LC_MESSAGES=en_US.UTF-8;LC_PAPER=en_US.UTF-8;LC_NAME=en_US.UTF-8;LC_ADDRESS=en_US.UTF-8;LC_TELEPHONE=en_US.UTF-8;LC_MEASUREME
2020-06-21 12:46:17: INFO[Main]: NT=en_US.UTF-8;LC_IDENTIFICATION=en_US.UTF-8
2020-06-21 12:46:17: VERBOSE[Main]: Determining world path
2020-06-21 12:46:17: INFO[Main]: Searching worlds...
2020-06-21 12:46:17: INFO[Main]:   In /home/johnathanr/.minetest/worlds: 
2020-06-21 12:46:18: INFO[Main]: Marshland Dolt Islands What the What synolvia-co world What Swan City Synolvia Co Chickens 
2020-06-21 12:46:18: INFO[Main]: 9 found.
2020-06-21 12:46:18: INFO[Main]: Creating default world at [/home/johnathanr/.minetest/worlds/world]
2020-06-21 12:46:18: VERBOSE[Main]: Determining gameid/gamespec
2020-06-21 12:46:18: INFO[Main]: Using world gameid [minetest]
2020-06-21 12:46:18: INFO[Main]: Using commanded world path [/home/johnathanr/.minetest/worlds/world]
AL lib: (EE) ALCpulsePlayback_connectStream: Stream did not get ready: Timeout
2020-06-21 12:48:05: ERROR[Main]: An unhandled exception occurred: Audio: Global Initialization: Context Create
2020-06-21 12:48:05: ERROR[Main]: In thread 7f49dfa34000:
2020-06-21 12:48:05: ERROR[Main]: /build/minetest-dHPfdb/minetest-5.2.0/src/main.cpp:e8: int main(int, char**): A fatal error occurred: Audio: Global Initialization: Context Create
Aborted (core dumped)
`
**Output of minetest --trace from video_driver = software**
```
2020-06-21 12:51:05: [Main]: Enabling trace level debug output
2020-06-21 12:51:05: INFO[Main]: Using system-wide paths (NOT RUN_IN_PLACE)
2020-06-21 12:51:05: INFO[Main]: Detected share path: /usr/share/minetest
2020-06-21 12:51:05: INFO[Main]: Detected user path: /home/johnathanr/.minetest
2020-06-21 12:51:05: INFO[Main]: Detected cache path: /home/johnathanr/.cache/minetest
2020-06-21 12:51:05: INFO[Main]: Using locale directory /usr/share/minetest/locale
2020-06-21 12:51:05: INFO[Main]: minetest with SER_FMT_VER_HIGHEST_READ=28, BUILD_TYPE=Release
2020-06-21 12:51:05: INFO[Main]: RUN_IN_PLACE=0
2020-06-21 12:51:05: INFO[Main]: USE_GETTEXT=1
2020-06-21 12:51:05: INFO[Main]: USE_SOUND=1
2020-06-21 12:51:05: INFO[Main]: USE_CURL=1
2020-06-21 12:51:05: INFO[Main]: USE_FREETYPE=1
2020-06-21 12:51:05: INFO[Main]: USE_LUAJIT=1
2020-06-21 12:51:05: INFO[Main]: STATIC_SHAREDIR="/usr/share/minetest"
2020-06-21 12:51:05: VERBOSE[Main]: httpfetch_init: parallel_limit=8
2020-06-21 12:51:05: INFO[Main]: Message locale is now set to: LC_CTYPE=en_US.UTF-8;LC_NUMERIC=C;LC_TIME=en_US.UTF-8;LC_COLLATE=en_US.UTF-8;LC_MONETARY=en_US.UTF-8;LC_MESSAGES=en_US.UTF-8;LC_PAPER=en_US.UTF-8;LC_NAME=en_US.UTF-8;LC_ADDRESS=en_US.UTF-8;LC_TELEPHONE=en_US.UTF-8;LC_MEASUREME
2020-06-21 12:51:05: INFO[Main]: NT=en_US.UTF-8;LC_IDENTIFICATION=en_US.UTF-8
2020-06-21 12:51:05: VERBOSE[Main]: Determining world path
2020-06-21 12:51:05: INFO[Main]: Searching worlds...
2020-06-21 12:51:05: INFO[Main]:   In /home/johnathanr/.minetest/worlds: 
2020-06-21 12:51:05: INFO[Main]: Marshland Dolt Islands What the What synolvia-co world What Swan City Synolvia Co Chickens 
2020-06-21 12:51:05: INFO[Main]: 9 found.
2020-06-21 12:51:05: INFO[Main]: Creating default world at [/home/johnathanr/.minetest/worlds/world]
2020-06-21 12:51:05: VERBOSE[Main]: Determining gameid/gamespec
2020-06-21 12:51:05: INFO[Main]: Using world gameid [minetest]
2020-06-21 12:51:05: INFO[Main]: Using commanded world path [/home/johnathanr/.minetest/worlds/world]
2020-06-21 12:52:48: INFO[Main]: Audio: Global Initialized: OpenAL 1.1 ALSOFT 1.18.2, using OpenAL Soft
2020-06-21 12:52:48: INFO[Main]: Irrlicht: Irrlicht Engine version 1.8.4
2020-06-21 12:52:48: INFO[Main]: Irrlicht: Linux 4.15.0-106-generic #107-Ubuntu SMP Thu Jun 4 11:27:52 UTC 2020 x86_64
2020-06-21 12:52:48: INFO[Main]: Irrlicht: Using plain X visual
2020-06-21 12:52:48: WARNING[Main]: Client: Could not find X11 Display in ExposedVideoData
2020-06-21 12:52:48: VERBOSE[Main]: Client: Configuring general top level window properties
2020-06-21 12:52:48: WARNING[Main]: Could not find x11 display for setting its icon.
2020-06-21 12:52:48: WARNING[Main]: Could not load icon file '/usr/share/minetest/misc/minetest-xorg-icon-128.png'
2020-06-21 12:52:48: VERBOSE[Main]: Client: Finished configuring general top level window properties
2020-06-21 12:52:48: INFO[Main]: text_height=20
2020-06-21 12:52:48: INFO[Main]: Waiting for other menus
2020-06-21 12:52:48: INFO[Main]: Waited for other menus
2020-06-21 12:52:48: INFO[Main]: Audio: Initialized: OpenAL 
2020-06-21 12:52:48: INFO[Main]: GUIEngine: Initializing Lua
2020-06-21 12:52:48: VERBOSE[AsyncWorker-0]: Loading and running script from /usr/share/minetest/builtin/init.lua
2020-06-21 12:52:48: VERBOSE[AsyncWorker-1]: Loading and running script from /usr/share/minetest/builtin/init.lua
2020-06-21 12:52:48: VERBOSE[AsyncWorker-2]: Loading and running script from /usr/share/minetest/builtin/init.lua
2020-06-21 12:52:48: VERBOSE[AsyncWorker-3]: Loading and running script from /usr/share/minetest/builtin/init.lua
2020-06-21 12:52:48: INFO[Main]: SCRIPTAPI: Initialized main menu modules
2020-06-21 12:52:48: VERBOSE[Main]: Loading and running script from /usr/share/minetest/builtin/init.lua
2020-06-21 12:52:48: INFO[AsyncWorker-3]: Initializing Asynchronous environment
2020-06-21 12:52:48: INFO[AsyncWorker-0]: Initializing Asynchronous environment
2020-06-21 12:52:48: INFO[AsyncWorker-1]: Initializing Asynchronous environment
2020-06-21 12:52:48: INFO[AsyncWorker-2]: Initializing Asynchronous environment
2020-06-21 12:52:50: INFO[Main]: Searching worlds...
2020-06-21 12:52:50: INFO[Main]:   In /home/johnathanr/.minetest/worlds: 
2020-06-21 12:52:50: INFO[Main]: Marshland Dolt Islands What the What synolvia-co world What Swan City Synolvia Co Chickens 
2020-06-21 12:52:50: INFO[Main]: 9 found.
Loaded texture: /usr/share/minetest/textures/base/pack/menu_bg.png
2020-06-21 12:52:51: VERBOSE[Main]: OpenALSoundManager::maintain(): 0 playing sounds, 0 sound names loaded
2020-06-21 12:52:51: INFO[Main]: Audio: Error opening /home/johnathanr/.minetest/sounds/main_menu.0.ogg for decoding
2020-06-21 12:52:51: INFO[Main]: Audio: Error opening /home/johnathanr/.minetest/sounds/main_menu.1.ogg for decoding
2020-06-21 12:52:51: INFO[Main]: Audio: Error opening /home/johnathanr/.minetest/sounds/main_menu.2.ogg for decoding
2020-06-21 12:52:51: INFO[Main]: Audio: Error opening /home/johnathanr/.minetest/sounds/main_menu.3.ogg for decoding
2020-06-21 12:52:51: INFO[Main]: Audio: Error opening /home/johnathanr/.minetest/sounds/main_menu.4.ogg for decoding
2020-06-21 12:52:51: INFO[Main]: Audio: Error opening /home/johnathanr/.minetest/sounds/main_menu.5.ogg for decoding
2020-06-21 12:52:51: INFO[Main]: Audio: Error opening /home/johnathanr/.minetest/sounds/main_menu.6.ogg for decoding
2020-06-21 12:52:51: INFO[Main]: Audio: Error opening /home/johnathanr/.minetest/sounds/main_menu.7.ogg for decoding
2020-06-21 12:52:51: INFO[Main]: Audio: Error opening /home/johnathanr/.minetest/sounds/main_menu.8.ogg for decoding
2020-06-21 12:52:51: INFO[Main]: Audio: Error opening /home/johnathanr/.minetest/sounds/main_menu.9.ogg for decoding
2020-06-21 12:52:51: INFO[Main]: Audio: Error opening /home/johnathanr/.minetest/sounds/main_menu.ogg for decoding
2020-06-21 12:52:51: INFO[Main]: Audio: Error opening /usr/share/minetest/sounds/main_menu.0.ogg for decoding
2020-06-21 12:52:51: INFO[Main]: Audio: Error opening /usr/share/minetest/sounds/main_menu.1.ogg for decoding
2020-06-21 12:52:51: INFO[Main]: Audio: Error opening /usr/share/minetest/sounds/main_menu.2.ogg for decoding
2020-06-21 12:52:51: INFO[Main]: Audio: Error opening /usr/share/minetest/sounds/main_menu.3.ogg for decoding
2020-06-21 12:52:51: INFO[Main]: Audio: Error opening /usr/share/minetest/sounds/main_menu.4.ogg for decoding
2020-06-21 12:52:51: INFO[Main]: Audio: Error opening /usr/share/minetest/sounds/main_menu.5.ogg for decoding
2020-06-21 12:52:51: INFO[Main]: Audio: Error opening /usr/share/minetest/sounds/main_menu.6.ogg for decoding
2020-06-21 12:52:51: INFO[Main]: Audio: Error opening /usr/share/minetest/sounds/main_menu.7.ogg for decoding
2020-06-21 12:52:51: INFO[Main]: Audio: Error opening /usr/share/minetest/sounds/main_menu.8.ogg for decoding
2020-06-21 12:52:51: INFO[Main]: Audio: Error opening /usr/share/minetest/sounds/main_menu.9.ogg for decoding
2020-06-21 12:52:51: INFO[Main]: Audio: Error opening /usr/share/minetest/sounds/main_menu.ogg for decoding
2020-06-21 12:52:51: INFO[Main]: OpenALSoundManager: "main_menu" not found.
Loaded texture: /usr/share/minetest/textures/base/pack/refresh.png
Loaded texture: /usr/share/minetest/textures/base/pack/blank.png
Loaded texture: /usr/share/minetest/textures/base/pack/server_flags_favorite.png
Loaded texture: /usr/share/minetest/textures/base/pack/server_ping_4.png
Loaded texture: /usr/share/minetest/textures/base/pack/server_ping_3.png
Loaded texture: /usr/share/minetest/textures/base/pack/server_ping_2.png
Loaded texture: /usr/share/minetest/textures/base/pack/server_ping_1.png
Loaded texture: /usr/share/minetest/textures/base/pack/server_flags_creative.png
Loaded texture: /usr/share/minetest/textures/base/pack/server_flags_damage.png
Loaded texture: /usr/share/minetest/textures/base/pack/server_flags_pvp.png
Segmentation fault (core dumped)
```
**Hypothesis of the issue**
Could it possibly be an error with pulseaudio?

**Comment by SmallJoker** [comment]
Thank you for these detailed logs. Looks like this time around it's related to OpenAL, and turns out to be a duplicate of #7250 .
Fixed by #9804 (5.3.0-dev).
